app/src/androidTest/java/ch/epfl/sdp/CreateEventFragmentTest.java

```diff
@@ -0,0 +1,100 @@
+package ch.epfl.sdp;
+
+import androidx.test.espresso.action.ViewActions;
+import androidx.test.rule.ActivityTestRule;
+import androidx.test.runner.AndroidJUnit4;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import ch.epfl.sdp.db.Database;
+import ch.epfl.sdp.ui.event.CreateEventFragment;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.clearText;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static androidx.test.espresso.action.ViewActions.scrollTo;
+import static androidx.test.espresso.action.ViewActions.typeText;
+import static androidx.test.espresso.matcher.ViewMatchers.withClassName;
+import static androidx.test.espresso.matcher.ViewMatchers.withHint;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.endsWith;
+import static org.hamcrest.Matchers.is;
+
+@RunWith(AndroidJUnit4.class)
+public class CreateEventFragmentTest {
+
+    private Database db = new MockDatabase();
+    private MockEvents mockEvents = new MockEvents();
+
+    @Rule
+    public final ActivityTestRule<MainActivity> mActivityRule =
+            new ActivityTestRule<>(MainActivity.class);
+
+    @Before
+    public void setup() {
+        onView(withText("Create")).perform(click());
```

Comment:
Parfois les tests peuvent foirer si tu fais juste ça: tu peux appeler dismissSystemPopups de TestUtils et ajouter .inRoot(withDecorView(is(mActivityRule.getActivity().getWindow().getDecorView()))) entre le onView et le perform